### PR TITLE
Add unit tests for wrapper handlers in tools/serve

### PR DESCRIPTION
**Pull request by qiuzhong**
Add unit tests for all kinds of wrapper handlers in tools/serve/serve.py.
See #12452

### CODE REVIEW

File: tools/wptserve/tests/functional/test_handlers.py [hunk 1]
```diff
@@ -8,8 +8,11 @@
 from six.moves.urllib.error import HTTPError
 
 wptserve = pytest.importorskip("wptserve")
-from .base import TestUsingServer, TestUsingH2Server, doc_root
+from .base import TestUsingServer, TestUsingH2Server, \
```
Comment (jgraham) [inline tools/wptserve/tests/functional/test_handlers.py]: Don't use continuation characters, please.

File: tools/wptserve/tests/functional/test_handlers.py [hunk 2]
```diff
@@ -388,5 +391,64 @@ def test_requesting_multiple_resources(self):
         assert resp.read() == ''
 
 
+class TestWorkersHandler(TestWrapperHandlerUsingServer):
+
```
Comment (jgraham) [inline tools/wptserve/tests/functional/test_handlers.py]: No blank line after the class definition.

**Review by jgraham — CHANGES_REQUESTED** [review]
This is amazing, thanks!

Only a couple of minor style fixes to request.

**Comment by sideshowbarker** [comment]
Travis unittest failures:

* https://travis-ci.org/web-platform-tests/wpt/jobs/452825483
* https://travis-ci.org/web-platform-tests/wpt/jobs/452825484

```
_________ ERROR collecting wptserve/tests/functional/test_handlers.py __________
ImportError while importing test module '/home/travis/build/web-platform-tests/wpt/tools/wptserve/tests/functional/test_handlers.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
.tox/py27/lib/python2.7/site-packages/six.py:709: in exec_
    exec("""exec _code_ in _globs_, _locs_""")
wptserve/tests/functional/test_handlers.py:14: in <module>
    from serve.serve import WorkersHandler, WindowHandler, AnyHtmlHandler,\
E   ImportError: No module named serve
```
and
```
ERROR: InvocationError for command '/home/travis/build/web-platform-tests/wpt/tools/.tox/py36/bin/pytest --cov' (exited with code 2)
py36-flake8 create: /home/travis/build/web-platform-tests/wpt/tools/.tox/py36-flake8
py36-flake8 installdeps: -rrequirements_flake8.txt
py36-flake8 installed: flake8==3.6.0,flake8-polyfill==1.0.2,mccabe==0.6.1,pep8-naming==0.7.0,pycodestyle==2.4.0,pyflakes==2.0.0
py36-flake8 run-test-pre: PYTHONHASHSEED='3166492845'
py36-flake8 runtests: commands[0] | flake8 --append-config=py36-flake8.ini
___________________________________ summary ____________________________________
ERROR:   py36: commands failed
```

**Comment by sideshowbarker** [comment]
See https://travis-ci.org/web-platform-tests/wpt/jobs/453768016; unittests still failing for w/ Python3

**Comment by qiuzhong** [comment]
@jgraham , thanks for your comments. I improved the code style.

@sideshowbarker , the import error and other errors have been fixed.

**Comment by sideshowbarker** [comment]
Yeah, c4401e444780e26141d44a7de90b30f2683e13fd is all green now, so when the review is approved, this should be ready to merge to master

**Review by gsnedders — APPROVED** [review]

**Review by jgraham — APPROVED** [review]

**Pull request merged**